bin/saya/src/args/mod.rs

```diff
@@ -73,6 +75,10 @@ pub struct SayaArgs {
     #[command(flatten)]
     #[command(next_help_heading = "Choose the proof pipeline configuration")]
     pub proof: ProofOptions,
+
```

Comment:
Would you mind also refactoring the prover options for the occasion? :) With `url` + `private_key`. Like this we can change enhance prover options later with less efforts.

Reply:
done